src/custom/components/SwapWarnings/index.tsx

```diff
@@ -39,15 +39,35 @@ const WarningContainer = styled(AuxInformationContainer).attrs((props) => ({
   ...props,
   hideInput: true,
 }))<HighFeeContainerProps>`
-  background: ${({ theme, bgColour }) => bgColour || theme.info};
-  color: ${({ theme, textColour }) => textColour || theme.infoText};
-  padding: ${({ padding = '10px' }) => padding};
+  --warningColor: ${({ theme, level }) =>
```

Comment:
Store color in a CSS VAR based on the `level` prop

Reply:
The props names are a bit strange. They should follow the pattern `<Name>Props`

In this case, I guess should be `WarningContainerProps` 

Same for `export const HighFeeWarning = (props: WarningProps) => {`

That one should be called `HighFeeWarningProps`